app/models/occupation_standard.rb

```diff
@@ -49,6 +51,36 @@ def clone_as_unregistered!(creator_id:, organization_id:)
     end
   end
 
+  def should_generate_attachment?(kind)
+    # Occupation standard updated_at timestamp gets set milliseconds after
+    # attachment created_at timestamp.
+    !send(kind).attached? || send(kind).created_at < updated_at - 1.second
+  end
+
+  def to_csv
+    CSV.generate do |csv|
+      csv << %w(rapids_code organization_title occupation_standard_title type work_process_title work_process_description work_process_hours work_process_sort skill skill_sort)
```

Comment:
Two things:

1. Let's include the `onet_code` here after `rapids_code`.

2. Just a style comment on this, but can we pull these headers out into a constant like 

```CSV_HEADERS = %w(rapids_code organization_title occupation_standard_title type work_process_title work_process_description work_process_hours work_process_sort skill skill_sort).freeze``` 

at the top of the model file?  Then we just have `csv << CSV_HEADERS` in the `#to_csv` method?